### PR TITLE
Remove snappy dependency

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by okkez** [comment]
See also https://twitter.com/repeatedly/status/743380575533207554

**Comment by tagomoris** [comment]
This change looks to need an update for README to mention how to install snappy.gem before using it.

**Comment by okkez** [comment]
I will update README :bow:

**Comment by okkez** [comment]
This PR needs #42

**Comment by okkez** [comment]
I will rebase this PR after merge #42

File: lib/fluent/plugin/webhdfs_compressor_snappy.rb [hunk 1]
```diff
@@ -4,7 +4,11 @@ class SnappyCompressor < Compressor
       WebHDFSOutput.register_compressor('snappy', self)
 
       def initialize(options = {})
-        require "snappy"
+        begin
+          require "snappy"
+        rescue LoadError
+          $log.error("Install snappy before use snappy compressor")
```
Comment (repeatedly) [inline lib/fluent/plugin/webhdfs_compressor_snappy.rb]: raising ConfigError is better.

**Comment by tagomoris** [comment]
LGTM.

**Comment by okkez** [comment]
Rebased

**Comment by tagomoris** [comment]
I missed to check CI result.
At v0.12 or ealier, `chunk.size` return bytesize. At v0.14, `chunk.size` returns the # of events in the chunk, not bytesize (and `chunk.bytesize` is added).
Could you update your tests like this?
```
chunk_bytesize = chunk.respond_to?(:bytesize) ? chunk.bytesize : chunk.size
assert(chunk_bytesize > io.read.bytesize)
```

**Comment by tagomoris** [comment]
Thank you!

**Comment by repeatedly** [comment]
Could you release new version?

**Comment by repeatedly** [comment]
@tagomoris ping.

**Comment by tagomoris** [comment]
Done > `0.5.2`. Sorry for missing this.

**Pull request merged**